doc/chapters/gateways/index.rst

```diff
@@ -302,12 +302,39 @@ An example configuration can look like this::
 Network gateway
 ===============
 
-The Network Gateway is used to provide Internet access through a container-specific firewall.
+The Network Gateway is used to setup network connection and configure which traffic is allowed and not.
 
 Configuration
 -------------
+Types: "INCOMING" and "OUTGOING"
+Targets: "ACCEPT", "DROP" and "REJECT"
+host: hostname or ip-address with netmask
+default: target to choose when not matching any other rule
 
-T.B.D.
+Example of network gateway config::
```

Comment:
Suggestion on collision resolution:
Merge the rules lists of all entries of same type using the most permissive default.

Example:
```json
 [
        {
            "type": "OUTGOING",
            "rules": [
                         { "host": "127.0.0.1/16", "port": 80, "target": "ACCEPT"},
                         { "host": "example.com", "port": "80-85", "target": "ACCEPT"}
                     ],
            "default": "REJECT"
        },
        {
            "type": "OUTGOING",
            "rules": [
                         { "host": "127.0.0.1/16", "port": [80, 8080], "target": "ACCEPT"},
                         { "host": "93.184.216.34/24", "target": "REJECT"}
                     ],
            "default": "DROP"
        }
    ]
```

Will become
```json
 [
        {
            "type": "OUTGOING",
            "rules": [
                         { "host": "127.0.0.1/16", "port": 80, "target": "ACCEPT"},
                         { "host": "example.com", "port": "80-85", "target": "ACCEPT"},
                         { "host": "127.0.0.1/16", "port": [80, 8080], "target": "ACCEPT"},
                         { "host": "93.184.216.34/24", "target": "REJECT"}
                     ],
            "default": "REJECT"
        }
    ]
```